### PR TITLE
Compile assets to a dist directory - Issue #251

### PR DESCRIPTION
**Pull request by castastrophe**
We do a lot of fussing to get around the fact that we're stuffing our compiled assets into the main directory of the component.  Wouldn't life be nice if we had a dist directory where we could go and find all our compiled assets? :)

### CODE REVIEW

**Comment by castastrophe** [comment]
TODO: 
[ ] Update all js paths
[ ] Update demo paths
[ ] Update storybook paths

**Comment by castastrophe** [comment]
@starryeyez024 Tagging you in this branch

File: elements/pfe-layouts/pfe-base.css [hunk 1]
```diff
@@ -0,0 +1,156 @@
+/*
```
Comment (starryeyez024) [inline elements/pfe-layouts/pfe-base.css]: why does this file appear in this MR?

Comment (castastrophe) [reply elements/pfe-layouts/pfe-base.css]: It was a leftover from migrating from compiling assets in root to compiling them into dist.  They no longer exist. :)

**Comment by castastrophe** [comment]
## Compile assets to a dist directory

### What has changed and why
Summarize files edited as part of this MR along with a brief description of what was changed/why.

* Assets get compiled from `_temp` to `dist`
* Assets specified in the `package.json` are available in the `dist` directory for each component
* All internal references (demo pages, etc.) have been updated

### Testing instructions
Be sure to include detailed instructions on how your update can be tested by another developer.

1. Compile the repository and make sure all demo pages work (`npm run demo`)
2. Preview storybook and ensure all storybook instances are rendering (`npm run storybook`)

#### Browser requirements
Your component should work in all of the following environments:

N/A

Your repository infrastructure updates should work for at least:
- [x] Node v8.x
- [x] NPM v7.x

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [ ] Was this feature demo'd and the design review approved?
- [ ] Did you update the CHANGELOG.md file with a summary of this update?


**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**

**Comment by starryeyez024** [comment]
When I run `npm run start` and `npm run dev` I get this error on pfe-layouts:
http://pastebin.test.redhat.com/799885

**Comment by starryeyez024** [comment]
@castastrophe http://localhost:8000/elements/pfe-layouts/pfe-layouts.min.css 404s in this branch

**Comment by castastrophe** [comment]
@starryeyez024 Just committed that update w00t thanks!

**Comment by starryeyez024** [comment]
Languid, Gelatinous, Tepid Mollusks.

👍

**Pull request merged**